ecosystem/typescript/sdk/src/aptos_types/authentication_key.ts

```diff
@@ -21,8 +21,12 @@ export class AuthenticationKey {
 
   static readonly ED25519_SCHEME: number = 0;
 
+  static readonly DERIVE_COLLECTION_OBJECT_ACCOUNT_SCHEME: number = 254;
```

Comment:
Let's call this DERIVE_OBJECT_FROM_SEED_SCHEME as this will later be used to derive token object address as well